README.md

```diff
@@ -319,6 +319,20 @@ template or not. If it's not, the CloudFormation custom resource
 templates described above and some validation/normalization features
 are disabled. Everything else should work as described above.
 
+### Converting Deployed Stacks to `iidy`
+Converting current stacks to `iidy` can be done throught the `iidy convert-stack-to-iidy` commad=nd,
+which accpets 2 parameters: `stackname` (the stack id of the project that you'd want to convert) and
```

Comment:
"accpets" -> "accepts"

I think you should say "stack name" instead of "stack id" (because stacks do have a UUID, and thats not what we're looking for here).